spec/features/posts_show_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.feature 'User Index Page', type: :feature do
+  before do
+    @user1 = User.create(name: 'Sam', photo: 'https://icons.iconarchive.com/icons/iconsmind/outline/256/User-icon.png',
+                         bio: 'Teacher from Poland.', posts_counter: 0)
+    visit users_path
+  end
+
+  scenario 'I can see the username, profile picture, and number of posts for each user' do
+    expect(page).to have_content(@user1.name)
+    expect(page.html).to include(@user1.photo)
+    expect(page).to have_content(@user1.posts_counter)
+  end
+
+  scenario 'When I click on a user, I am redirected to their show page' do
+    click_on @user1.name
+    expect(current_path).to eq(user_path(@user1))
+  end
```

Comment:
- [ ] kindly add the following tests i.e `Tests that asserts you can see the post's title.`, `Tests that asserts you can see who wrote the post.`, `Tests that asserts you can see how many comments it has.`, `Tests that asserts you can see how many likes it has.`, `Tests that asserts you can see the post body.`, `* Tests that asserts you can see the username of each commenter. Tests that asserts you can see the comment each commentor left.`

Reply:
Added!
![image](https://github.com/maov19/blog-app/assets/108192399/ca253bef-24dd-4737-9257-6e3e4574ab6d)